installer/hifi_light.nsi

```diff
@@ -565,13 +567,31 @@
 ; START Step 4:
 ; Launch Interface with command-line arguments
 ;--------------------------------
+    Var InterfaceCommandArgs
+    
+    Var MorphAvatarFile
     Function LaunchInterface
         ;MessageBox MB_OK "$HiFiInstalled"
+        ${StrContains} $StrContainsResult "/forceNoLaunchClient" $CMDLINE
         ${If} $HiFiInstalled == "true"
+        ${AndIf} $StrContainsResult == ""
             ; Make sure that no High Fidelity application is already running
             !insertmacro CheckForRunningApplications
             Call GetInterfacePath ;; In case it changed during installation of a new version
-            Exec '"$InterfacePath" --url "${EVENT_LOCATION}" --suppress-settings-reset --skipTutorial --cache "${ContentPath}\Interface" --scripts "${ContentPath}\Interface\scripts"'
+            StrCpy $InterfaceCommandArgs '--url "${EVENT_LOCATION}"  --suppress-settings-reset --skipTutorial --cache "${ContentPath}\Interface" --scripts "${ContentPath}\Interface\scripts"'
+
+            ; Demonstrate that we can pick up the beta RR avatar from file.
+            StrCpy $MorphAvatarFile "$AppData\..\LocalLow\Morph3D\ReadyRoom\High_Fidelity_RR_Launch.js"
```

Comment:
I assumed AppData couldn't be used in a define, but since you say it can, I'll change it.